weTranslate/NoResultView.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  NoResultTableViewCell.swift
+//  weTranslate
+//
+//  Created by Lionel on 2/28/16.
+//  Copyright © 2016 weTranslate. All rights reserved.
+//
+
+import UIKit
+
+final class NoResultView: UIView {
+
+    // MARK: - Properties
+
+    var viewModel: NoResultViewModel? {
+        didSet {
+            noResultLabel.text = viewModel?.message
+            emojiLabel.text = viewModel?.emoji
```

Comment:
could NoResultViewModel be a non optional ?
Also viewModel could be static instead of var ?